packages/auth-construct/API.md

```diff
@@ -84,6 +85,12 @@ export type CustomAttributeString = CustomAttributeBase & StringAttributeConstra
     dataType: 'String';
 };
 
+// @public
+export type CustomEmailSenderConstruct = {

```

Comment:
```suggestion
export type CustomEmailSender = {

```

or `CustomEmailSenderOptions`

This is not a construct. It has a reference to IFunction inside but this type is not implementing `Construct` super class.